src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -10,48 +10,97 @@
 
 namespace System
```

Comment:
Yes, I generated it using GetAPI as you suggested before. next time I'll touch this, I'll regenerate it using the target you have pointed at.